site/src/pages/chapters.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ChaptersPage from "../sections/Chapters";
+
+const Chapters = () => {
+  return (
+    <>
+      <ChaptersPage />
+    </>
```

Comment:
Add the `SeoComponent`, `Layout` and the `Footer` similar to what we have in the [index](https://github.com/service-mesh-patterns/service-mesh-patterns/blob/master/site/src/pages/index.js) file

Reply:
Do I need to include the schema from the index page as well?